### PR TITLE
Feature/users pagination

### PR DESCRIPTION
**Pull request by Ravennod**

### CODE REVIEW

File: MedicalDocManagment.WebUI/Controllers/api/AdminController.cs [hunk 1]
```diff
@@ -28,6 +31,20 @@ public IHttpActionResult GetUsers()
         }
 
         [HttpGet]
+        public IHttpActionResult GetPaged(int pageNumber = 1, int pageSize = 20)
+        {
+            int skip = (pageNumber - 1) * pageSize;
+            int total = UsersManager.Users.Count();
+            var users = UsersManager.Users
+                      .OrderBy(c => c.Id)
+                      .Skip(skip)
```
Comment (artempanko) [inline MedicalDocManagment.WebUI/Controllers/api/AdminController.cs]: please format

File: MedicalDocManagment.WebUI/Helpers/PagedResultHelper.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        public PagedResultHelper(IEnumerable<T> items, int pageNumber, int pageSize, long totalRecordCount)
+        {
+            Data = new List<T>(items);
+            Paging = new PagingInfo
+            {
+                PageNumber = pageNumber,
+                PageSize = pageSize,
+                TotalRecordCount = totalRecordCount,
+                PageCount = totalRecordCount > 0
+                          ? (int)Math.Ceiling(totalRecordCount / (double)pageSize)
```
Comment (artempanko) [inline MedicalDocManagment.WebUI/Helpers/PagedResultHelper.cs]: please format

File: MedicalDocManagment.WebUI/Helpers/PagedResultHelper.cs [hunk 3]
```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace MedicalDocManagment.WebUI.Helpers
+{
+    public class PagedResultHelper<T>
+    {
+        public List<T> Data { get; private set; }
```
Comment (artempanko) [inline MedicalDocManagment.WebUI/Helpers/PagedResultHelper.cs:10]: Bad variable name

**Review by artempanko — APPROVED** [review]
With minor comments

**Review by KostetskyiYaroslav — APPROVED** [review]

File: MedicalDocManagment.WebUI/Helpers/PagedResultHelper.cs (hunk 3 above)
Comment (Ravennod) [reply MedicalDocManagment.WebUI/Helpers/PagedResultHelper.cs:10]: This helper should be used not only for users pagination, so it's ok.

**Pull request merged**